3.12/graphs-general-graphs.md

```diff
@@ -7,16 +7,60 @@ General Graphs
 ==============
 
 This chapter describes the [general-graph](graphs.html) module.
-It allows you to define a graph that is spread across several edge and document collections.
-This allows you to structure your models in line with your domain and group them logically in collections giving you the power to query them in the same graph queries.
-There is no need to include the referenced collections within the query, this module will handle it for you.
+It allows you to define a graph that is spread across several edge and document
+collections.
+This allows you to structure your models in line with your domain and group
+them logically in collections giving you the power to query them in the same
+graph queries.
+There is no need to include the referenced collections within the query, this
+module will handle it for you.
 
 New to ArangoDB? Take the free
 [ArangoDB Graph Course](https://www.arangodb.com/arangodb-graph-course){:target="_blank"}
 for freshers.
 
-Three Steps to create a graph
------------------------------
+## Getting started
+
+### Create a GeneralGraph using the Web Interface
+
+The Web Interface (also called Web UI) allows you to easily create and manage
+GeneralGraphs. To get started, follow the steps outlined below.
+
+1. In the main page of the Web Interface, go to the left sidebar 
+   menu and select the **Graphs** tab.
+2. To add a new graph, click **Add Graph**.
+3. In the **Create Graph** dialog that appears, select the
+   **GeneralGraph** tab.
+4. Fill in the following fields:
+   - For **Name**, enter a name for the GeneralGraph.
+   - For **Shards**, enter the number of shards the graph is using.
```

Comment:
should be "number of shards to use", or maybe even "number of parts to split the graph into"

Reply:
Done.